### PR TITLE
Refactored Accelerator Config for WarpScript lib compatibility

### PR DESCRIPTION
**Pull request by hbs**

### CODE REVIEW

File: warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  
+  public static void setChunkCount(int count) {
+    chunkCount = count;
+  }
+  
+  public static void setChunkSpan(long span) {
+    chunkSpan = span;
+  }
+  public static int getChunkCount() {
+    return 0;
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java:135]: `return chunkCount`

File: warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  public static long getChunkSpan() {
+    return 0L;
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java:138]: `return chunkSpan`

File: warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  public static final String NOCACHE = "nocache";
+  public static final String NOPERSIST = "nopersist";
+  public static final String CACHE = "cache";
+  public static final String PERSIST = "persist";
+  public static final String ACCELERATOR_HEADER = "X-Warp10-Accelerator";
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java:144]: I would have put these constant as the beginning of the class.

File: warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  public static final boolean accelerated() {
+    if (instantiated) {
+      return accelerated.get();
+    } else {
+      return false;
+    }
+  }
+  public static final String ATTR_NOCACHE = "accel.nocache";
+  public static final String ATTR_NOPERSIST = "accel.nopersist";
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java:121]: I would have put these constant as the beginning of the class.

**Comment by hbs** [comment]
Bloody refactoring....

PTAL

File: warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+      nocache.set(Boolean.FALSE);
+    }
+  }
+  
+  public static final void nopersist() {
+    if (instantiated) {
+      nopersist.set(Boolean.TRUE);
+    }
+  }
+  public static final void persist() {
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java:111]: A an empty line between these methods.

File: warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+  }
+  
+  public static void setChunkSpan(long span) {
+    chunkSpan = span;
+  }
+  
+  public static int getChunkCount() {
+    return chunkCount;
+  }
+  public static long getChunkSpan() {
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/AcceleratorConfig.java:156]: A an empty line between these methods.

**Pull request merged**